gc/base/standard/ConcurrentGC.cpp

```diff
@@ -2209,6 +2239,8 @@ MM_ConcurrentGC::signalThreadsToActivateWriteBarrier(MM_EnvironmentBase *env)
 			reportGCCycleStart(env);
 			env->_cycleState = previousCycleState;
 
+			_concurrentPhaseStats.clear();
+			preConcurrentInitializeStatsAndReport(env);
```

Comment:
I'm deviating a little from the pattern that MainGCThread has (of clearing stats and calling  `preConcurrentInitializeStatsAndReport`) - I personally like clearing stats inside the method, but feel free to leave it as is